docs/src/implemented-proposals/rent.md

```diff
@@ -14,6 +14,14 @@ Currently, the rent cost is fixed at the genesis. However, it's anticipated to b
 
 There are two timings of collecting rent from accounts: \(1\) when referenced by a transaction, \(2\) periodically once an epoch. \(1\) includes the transaction to create the new account itself, and it happens during the normal transaction processing by the bank as part of the load phase. \(2\) exists to ensure to collect rents from stale accounts, which aren't referenced in recent epochs at all. \(2\) requires the whole scan of accounts and is spread over an epoch based on account address prefix to avoid load spikes due to this rent collection.
 
+On the contrary, rent collection isn't applied to accounts that are directly manipulated by any of protocol-level bookkeeping processes including:
+
+- The distribution of rent collection itself (Otherwise, it may cause recursive rent collection handling)
+- The distribution of staking rewards at the start of every epoch (To reduce as much as processing spike at the start of new epoch)
+- The distribution of transaction fee at the end of every epoch
+
+Even if those processes are out of scope of rent collection, all of manipulated accounts will eventually be handled by the \(2\) mechanism.
+
```

Comment:
This sounds good to me.  It'd be great to get a comment from @rwalker-com to be sure we aren't missing anything. :pray:

Reply:
lgtm, and this is consistent with what I intended, but the call flow to load and store accounts is pretty tortured, and has many paths.  were the bank to document which APIs were for runtime access of accounts, it would go a long way to making these easier to reason about.

Reply:
> the call flow to load and store accounts is pretty tortured ...

@rwalker-com Yeah, I think so too... I'll address the documetation work along with #10426

Anyway, thanks for confirming this pr!

Reply:
Err, wrong user @rwalker-apple...